src/pages/services/aem-assets-view/api/browse-view/index.md

```diff
@@ -0,0 +1,497 @@
+---
+title: Browse View - AEM Assets View Extensibility
+description: Explore the ways to extend and customize Browse View in AEM Assets View
+contributors:
+- https://github.com/AdobeDocs/uix
+---
+
+# Browse View
+
+AEM Assets View offers the ability to customize the ActionBar and QuickActions in the Browse View.
+
+The Browse View in the AEM Assets View refers to the asset listing pages such as
+Assets, Collections, Recent, Search and Trash.
+
+**ActionBar** is the blue bar with actions that appears at the top when one or more assets in the
+Browse View are selected.
+
+![action bar](action-bar.png)
+
+**QuickActions** is the dropdown menu from the More action button (shown as `⋯`) next to each asset.
+![quick actions](quick-actions.png)
+
+Extensions should use the `aem/assets/browse/1` extension point to utilize extensibility services of the Browse View.
+
+An extension needs to implement both `actionBar` and `quickActions` namespace to be recognized by Assets View.
+
+## Custom ActionBar actions and QuickActions menu actions
+
+This extensibility feature allows context-aware customization of the ActionBar actions and the QuickActions menu actions
+associated with the selected resources.
+
+Using the `actionBar` namespace, custom actions could be added to the ActionBar after the list of built-in actions, and
+the built-in actions could be overridden or hidden based on the context and the selected resources.
+
+In this example, a custom action is added to the ActionBar after the list of built-in ActionBar actions.
+
+![ActionBar actions](action-bar-action.png)
+
+Using the `quickActions` namespace, built-in QuickActions menu actions can be overridden and hidden based on the context and the
+selected asset.
+
+## API Reference
+
+This API reference section is further broken down into two parts: the API provided by the AEM Assets View host application
+to the extension and the API provided by the extension to the AEM Assets View host application.
+
+### Host API Reference
+
+In addition to the [Common API](../commons) provided by AEM Assets View to all extensions,
+the host application provides the following definitions that are specific to the `aem/assets/browse/1` extension point,
+the `actionBar` and `quickActions` namespaces.
+
+#### Browsing context
+
+Assets View supports assets browsing experiences in multiple modes, or "contexts". The current context is exposed in
+the Host API and is also communicated to Extension APIs, so that the custom code within Extension can adapt to the
+state of Assets View.
+
+The supported browsing contexts are:
+
+| Browsing Context | Description | Path |
+|------------|------------|------------|
+| assets | Main asset browsing experience | /assets/browse |
```

Comment:
Let's surround values in Browsing Context column with backticks as they are JS IDs.